fastn-core/src/auth/mod.rs

```diff
@@ -2,17 +2,40 @@ pub(crate) mod config;
 pub(crate) mod github;
 pub(crate) mod routes;
 
+mod emailpassword;
+
 mod utils;
 
-pub use utils::{decrypt, encrypt};
+use std::str::FromStr;
+
+pub const COOKIE_NAME: &str = "session";
+
+#[derive(Debug, serde::Deserialize, serde::Serialize)]
+pub struct FastnUser {
+    pub username: String,
+    pub id: uuid::Uuid,
+    pub name: Option<String>,
+    pub email: Option<String>,
```

Comment:
Why is email optional?

Reply:
I have made changes. Now these are not optional and `id` is not `uuid`.